app/views/tag/_location.html.erb

```diff
@@ -15,7 +15,7 @@
       <% end %>
       if( $(this).attr("submit_to") ) {
         modalUrl += firstVar ? '?' : '&';
-        modalUrl += 'url=' + encodeURIComponent(locationInput.attr("submit_to"));
+        modalUrl += 'url=' + encodeURIComponent($(this).attr("submit_to"));
```

Comment:
What is `this` in this context? 
`locationInput ` made more sense, right?

Reply:
I think $(this) refers to the location button
![Screenshot 2021-02-02 at 19 52 42](https://user-images.githubusercontent.com/7622875/106633893-4c04a180-6590-11eb-9b08-c0300b8e2888.png)
![Screenshot 2021-02-02 at 19 52 55](https://user-images.githubusercontent.com/7622875/106633901-4dce6500-6590-11eb-9525-6cb1eea60e44.png)